tests/templates/script/commonPartials.js

```diff
@@ -0,0 +1,62 @@
+const path = require('path');
+const fs = require('fs');
+const hbs = require('../../test-utils/hbs');
+const directoryPath = 'templates/common-partials/script'
+const partialPath = path.resolve(__dirname, `../../../${directoryPath}`);
+const files = fs.readdirSync(partialPath);
+
+/*
+//Generates common-partials default values
+files.forEach(function (file) {
+    const filename = file.concat('.js')
+    var tempPath = path.join(__dirname, "fixtures", filename);
+    fs.writeFile(tempPath, hbs.compile(fs.readFileSync(path.join(partialPath, file), 'utf-8'))({
+            "verticalKey": "testKey",
+            "componentSettings": {
+                "AppliedFilters": {"a": "testAF"},
+                "Facets": {"a":"testF"},
+                "FilterBox": {"a":"testFB"},
+                "LocationBias": {"a":"testLB"},
+                "Navigation": {"a":"testN"},
+                "QASubmission": {"a":"testQAS"},
+                "SearchBar": {"a":"testSB"},
+                "SortOptions": {"a":"testSO"},
+                "SpellCheck": {"a":"testSC"},
+                "VerticalResults": {"a":"testVR"},
+                "VerticalResultsCount": {"a":"testVRC"}
+            }
+        }), err => {
+        if (err) {
+            console.error(err)
+            return
+        }
+    });
+});
+*/
+
+
+it('checks all common partials', () => {
```

Comment:
If we do the for loop outside of the `it`, we can get N tests with 1 expect each, instead of 1 test with N expects. The issue with 1 test with N expects is that it will short circuit on the first expect that breaks. With N tests, it's also clearer exactly which partial broke. You can use a template string for the name of the test to insert the name of the partial being tested https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals